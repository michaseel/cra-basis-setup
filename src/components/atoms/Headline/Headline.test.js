import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import Headline from './Headline';

it('renders Headline snapshot', () => {
  const tree = renderer.create(<Headline>Test</Headline>).toJSON();
  expect(tree).toMatchStyledComponentsSnapshot();
});
