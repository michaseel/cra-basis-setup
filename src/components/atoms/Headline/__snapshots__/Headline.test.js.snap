// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Headline snapshot 1`] = `
.ccXnHE {
  font-size: 1.5em;
  -webkit-text-align: center;
  text-align: center;
  color: #000;
}

<h1
  className="sc-bdVaJa ccXnHE"
>
  Test
</h1>
`;
